superset/cachekeys/api.py

```diff
@@ -40,10 +42,98 @@ class CacheRestApi(BaseSupersetModelRestApi):
     class_permission_name = "CacheRestApi"
     include_route_methods = {
         "invalidate",
+        "warm_up_cache",
     }
 
     openapi_spec_component_schemas = (CacheInvalidationRequestSchema,)
 
+    @expose("/warm_up_cache", methods=["GET"])
+    @protect()
+    @safe
+    @statsd_metrics
+    @event_logger.log_this_with_context(
+        action=lambda self, *args, **kwargs: f"{self.__class__.__name__}"
+        f".warm_up_cache",
+        log_to_statsd=False,
+    )
+    def warm_up_cache(self) -> Response:
+        """Warms up the cache for the slice or table.
+
+        Note for slices a force refresh occurs.
+
+        In terms of the `extra_filters` these can be obtained from records in the JSON
+        encoded `logs.json` column associated with the `explore_json` action.
+
+        ---
+        get:
+          description: >-
```

Comment:
nit: let's start using summary instead, place the method docs into the OAS description